src/core/image/image.controller.ts

```diff
@@ -5,8 +5,20 @@ import { createCropHandler } from './resize/crop.handler';
 import { createThumbnailHandler } from './resize/thumbnail.handler';
 
 export const imageController = async (fastify: FastifyInstance) => {
-  createMetadataHandler('/metadata', fastify);
-  createThumbnailHandler('/thumbnail', fastify);
-  createCropHandler('/:adapter/crop', fastify);
-  createResizeHandler('/:adapter/resize', fastify);
+  const schema = {
```

Comment:
For me, it is parent schema or base schema. Just schema means nothing.

Reply:
fixed